client/helpers.go

```diff
@@ -121,7 +121,12 @@ func changeTargetsDelegation(repo *tuf.Repo, c changelist.Change) error {
 			return err
 		}
 		r.RemoveKeys(td.RemoveKeys)
-		r.RemovePaths(td.RemovePaths)
+
+		removePaths := td.RemovePaths
+		if td.ClearPaths {
```

Comment:
Not sure if I'm convinced of the need of adding a new `bool`. What if we iterate through `removePaths`, and if we find a `""`, we simply remove all of them?